.flake8

```diff
@@ -0,0 +1,18 @@
+[flake8]
```

Comment:
We already have some flake8 configuration in setup.cfg, I'm not sure what's preferable between setup.cfg and .flake8 these days